hosting/nginx.prod.conf.hbs

```diff
@@ -126,6 +130,25 @@ http {
       proxy_pass      http://$apps:4002;
     }
 
+    location /api/webhooks/ {
```

Comment:
Open to suggestions on how to reduce duplication between this location and the above `location /api/`. Nesting doesn't seem to work.

Reply:
You can nest locations in NGINX, but I think it's honestly just cleaner here to have the other route.

Reply:
I would agree, duplication in this case is probably easier to understand from a readability point of view.